sdk/src/test/java/io/opentelemetry/sdk/metrics/MeterSdkTest.java

```diff
@@ -54,19 +53,15 @@ public void testLabelSets() {
     MeterSdk testSdk = new MeterSdk();
 
     assertThat(testSdk.emptyLabelSet()).isSameInstanceAs(testSdk.emptyLabelSet());
+    assertThat(testSdk.emptyLabelSet()).isSameInstanceAs(testSdk.createLabelSet());
     assertThat(testSdk.emptyLabelSet())
         .isSameInstanceAs(testSdk.createLabelSet(Collections.<String, String>emptyMap()));
+
     assertThat(testSdk.createLabelSet("key", "value"))
         .isEqualTo(testSdk.createLabelSet("key", "value"));
-    assertThat(testSdk.createLabelSet("k1", "v1", "k2", "v2"))
```

Comment:
Why are these removed?

Reply:
Hm. I guess having at least one of more than a single pair would be useful. Beyond that, it seems like the value will be very low, since varargs is varargs. I'll put the 2-key version back.

Reply:
done